### PR TITLE
task(issue-84): added comments for inventory controllers 

### PR DESCRIPTION
**Pull request by mlee97**
Added descriptive comments for the controllers used in the inventory page.

### CODE REVIEW

File: ERP/app/Http/Controllers/BikeController.php [hunk 1]
```diff
@@ -17,9 +17,15 @@
...
+    /**
+    * Creates a custom bicycle as specified by the user from a form.
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function createBike(Request $request){
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
         $validator = Validator::make($request->all(), [
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/BikeController.php:29]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/BikeController.php [hunk 2]
```diff
@@ -55,21 +65,30 @@ public function createBike(Request $request){
...
+    /**
+    * Edits the specified bicycle from the bikes table
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function editBike(Request $request) {
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
         $validator = Validator::make($request->all(), [
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/BikeController.php:92]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/MaterialController.php [hunk 3]
```diff
@@ -11,51 +11,74 @@
...
+    /**
+    * Creates a custom material as specified by the user from a form.
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function createMaterial(Request $request){
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
         $validator = Validator::make($request->all(), [
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/MaterialController.php:23]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/MaterialController.php [hunk 4]
```diff
@@ -11,51 +11,74 @@
...
-    //
+    /**
+    * Creates a custom material as specified by the user from a form.
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function createMaterial(Request $request){
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/MaterialController.php:22]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/MaterialController.php [hunk 5]
```diff
@@ -64,41 +87,57 @@ public function destroy($id, Request $request) {
...
 
+    /**
+    * Edits the specified material from the materials table
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
      public function editMaterial(Request $request) {
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/MaterialController.php:104]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/PartController.php [hunk 6]
```diff
@@ -11,50 +11,74 @@
...
-    //
+    /**
+    * Creates a custom part as specified by the user from a form.
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function createPart(Request $request){
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/PartController.php:22]: You have an extra "the" after "Validate that..."

File: ERP/app/Http/Controllers/PartController.php [hunk 7]
```diff
@@ -63,42 +87,56 @@ public function destroy($id, Request $request) {
...
+    
+    /**
+    * Edits the specified part from the parts table
+    *
+    * @param  $request  stores the request method and its inputs, cookies, and files that were submitted with the request.
+    * @return view      the inventory view.
+    */
     public function editPart(Request $request) {
 
+        //Validate that the all the following fields are specified and that they match their respective data type.
```
Comment (CeliaCaii) [inline ERP/app/Http/Controllers/PartController.php:104]: You have an extra "the" after "Validate that..."

**Review by CeliaCaii — CHANGES_REQUESTED** [review]
There are recurring typos to be fixed. Otherwise, good work!!!

**Review by CeliaCaii — APPROVED** [review]

**Review by pirasunt — APPROVED** [review]

**Pull request merged**